### ISSUE TITLE
suppress CLI output on success

### ISSUE THREAD

**Issue by jonathantneal** [postcss/postcss-cli#98]
It would be nice to provide an option to surpress cli output on success.

Like `--silent`

Related: there is presently no option to disable the red logo.

**Comment by RyanZim** [comment]
As mentioned in gitter, I'm not opposed to this. However, the v3.0.0 release is feature frozen. Definitely would like to see this for v3.1.0, though.

**Comment by michael-ciniawsky** [comment]
@RyanZim Instead of an extra option maybe better disable it enterily and show it only for `--help|-h` && `--version]-v`. Just remove the `console.warn(chalk.bold.red(${logo}))` line, could be added to #97 ? 😛

**Comment by michael-ciniawsky** [comment]
Indicating success should stay imho, maybe it's possible to update the spinner per file and print it only once in watch mode, so no new lines are added on foreach success. I will take a look at it later...

**Comment by RyanZim** [comment]
> Just remove the console.warn(chalk.bold.red(${logo})) line

Done via a direct push to `develop`.

**Comment by RyanZim** [comment]
Attn: @jonathantneal @ben-eb @ihappycoder

OK, in the next release, the logo will not print during normal usage. The `✔ Finished base.css (20ms)` messages, etc. will still be displayed. 

Is that satisfactory, or do you still want a quiet flag to silence all output (except errors)? Personally, I like verbose tools, but YMMV.

**Comment by ben-eb** [comment]
@RyanZim I think that's fine, in this case you are displaying helpful information to the user. 👍

**Comment by Ryuno-Ki** [comment]
Personally I'd like to have a way to mute [that output](https://github.com/postcss/postcss-cli/blob/v3.0.0-beta/index.js#L305) as well. Otherwise I need the ability to send STDOUT to `/dev/null` cross-platform.

I've checked that I can still see error logs.

Background: We use a custom postcss plugin which does linting and want to be able to get error reports. Outputting the CSS files just adds too much noise.

**Comment by RyanZim** [comment]
@Ryuno-Ki Valid point. Could you please open a separate issue for that?

**Comment by RyanZim** [comment]
v3.0.0 is shipped! :tada: Take it for a spin and see if you like the new output.

**Comment by RyanZim** [comment]
Fixed in v5.
